packages/expect/src/__tests__/spyMatchers.test.js

```diff
@@ -870,197 +870,203 @@ const jestExpect = require('../');
 
     const basicReturnedWith = ['toHaveReturnedWith', 'toReturnWith'];
     if (basicReturnedWith.indexOf(returnedWith) >= 0) {
-      test(`works with more calls than the limit`, () => {
```

Comment:
ignore whitespace here (jest eslint complained about duplicate test names (now discovered due to https://github.com/jest-community/eslint-plugin-jest/pull/239))